OpenCover.UI.TestDiscoverer/NUnit/NUnitDiscoverer.cs

```diff
@@ -111,7 +135,11 @@ private TestMethod[] DiscoverTestsInClass(TypeDefinition type, TestClass @class)
 			foreach (var method in type.Methods)
 			{
 				bool isTestMethod = false;
-				var trait = new List<string>();
+                var traits = new List<string>();
+                if (@class.Traits != null && @class.Traits.Length > 0)
+                {
+                    traits.AddRange(@class.Traits);
+                }
```

Comment:
I have made the changes per your suggestion and started adding some tests to exercise the new functionality but got stuck.  I've never worked on a project that had a mixture of MSTest and NUnit test and can't figure out how to get the new tests to be found and how to execute them.  Do you think you can give me an overview of how you run tests, and any special considerations writing them?  Thanks!